Sources/ScrollingTabController.swift

```diff
@@ -202,18 +196,32 @@ open class ScrollingTabController: UIViewController, UIScrollViewDelegate, UICol
     }
 
     func configureViewControllers() {
-        for item in items {
```

Comment:
So I think if we remove this, then you can't just reset the list of view controllers. Which I think should also be possible / allowed